python/res/job_queue/workflow_runner.py

```diff
@@ -2,6 +2,20 @@
 from res.job_queue import Workflow
 from res.util.substitution_list import SubstitutionList
 
+class WorkflowRunnerThread(Thread):
+    def __init__(self, target, name):
+        super(WorkflowRunnerThread, self).__init__(target=target, name=name)
+        self.setDaemon(True)
+        self._exception = None
+        # in python2.7 it's called __target therefore we need to inject this too
+        self._target = target
+
+    def run(self):
+        try:
+            if self._target:
```

Comment:
Should this not be done by `super` instead? I.e. `super(WorkflowRunnerThread, self).run(self)`?

Reply:
Let's discuss this - need to know an opinion on this one